Model in custom list results in error

**Describe the bug**
Having a model on the latest version (0.144.0) in a custom list results in a 500 error
Liking a model works fine

**To Reproduce**
Steps to reproduce the behavior:
1. Add a model to a custom list
2. Save & reload page
3. See Internal Server error / 500 error

**Expected behavior**
Well, don't break 😂

Forgot to add the log

```
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] ActionView::Template::Error (undefined method 'name' for an instance of Components::ListsCard)
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] Caused by: NoMethodError (undefined method 'name' for an instance of Components::ListsCard)
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] 
23:12:53 rails.1              | Information for: ActionView::Template::Error (undefined method 'name' for an instance of Components::ListsCard):
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     144:         <% end %>
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     145:       <% end %>
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     146: 
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     147:       <%= ListsCard(listable: @model) if policy(:list).show? %>
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     148: 
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     149:       <%= SourcesCard(model: @model) %>
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]     150:       <%= RemixesCard(model: @model) %>
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]   
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:81:in 'block (4 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:81:in 'block (3 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:80:in 'block (2 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:79:in 'Array#map'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:79:in 'block in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:78:in 'Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:24:in 'block (2 levels) in Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:23:in 'block in Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:17:in 'block (2 levels) in Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:16:in 'block in Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:14:in 'Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:22:in 'Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/helpers/components_helper.rb:7:in 'block (2 levels) in <module:ComponentsHelper>'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/views/models/show.html.erb:147
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/controllers/application_controller.rb:141:in 'ApplicationController#switch_locale'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] 
23:12:53 rails.1              | Information for cause: NoMethodError (undefined method 'name' for an instance of Components::ListsCard):
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85]   
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:81:in 'block (4 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:81:in 'block (3 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:80:in 'block (2 levels) in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:79:in 'Array#map'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:79:in 'block in Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:78:in 'Components::ListsCard#on_lists'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:24:in 'block (2 levels) in Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:23:in 'block in Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:17:in 'block (2 levels) in Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:16:in 'block in Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/card.rb:14:in 'Components::Card#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/components/lists_card.rb:22:in 'Components::ListsCard#view_template'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/helpers/components_helper.rb:7:in 'block (2 levels) in <module:ComponentsHelper>'
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/views/models/show.html.erb:147
23:12:53 rails.1              | [d2f7c33a-9a6d-45be-8b06-9699eba2cd85] app/controllers/application_controller.rb:141:in 'ApplicationController#switch_locale'
```

--- a/app/components/lists_card.rb
+++ b/app/components/lists_card.rb
@@ -76,11 +76,11 @@ class Components::ListsCard < Components::Base
   def on_lists
     span { t("components.lists_card.on_lists") }
     table class: "table table-striped" do
-      @on_lists.map do
+      @on_lists.map do |list|
         tr do
-          td { link_to it.name, list_path(it) }
+          td { link_to list.name, list_path(list) }
           td do
-            link_to remove_path(list: it), method: :patch, class: "link-danger link-underline-opacity-0" do
+            link_to remove_path(list: list), method: :patch, class: "link-danger link-underline-opacity-0" do
               Icon(icon: "trash", label: t("components.lists_card.remove"))
             end
           end

--- a/app/views/robots/sitemap.xml.builder
+++ b/app/views/robots/sitemap.xml.builder
@@ -5,22 +5,22 @@ xml.urlset "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
   xml.url { xml.loc creators_url }
   xml.url { xml.loc collections_url }
   xml.url { xml.loc models_url }
-  @creators.each do
+  @creators.each do |creator|
     xml.url do
-      xml.loc creator_url(it)
-      xml.lastmod it.updated_at.iso8601
+      xml.loc creator_url(creator)
+      xml.lastmod creator.updated_at.iso8601
     end
   end
-  @collections.each do
+  @collections.each do |collection|
     xml.url do
-      xml.loc collection_url(it)
-      xml.lastmod it.updated_at.iso8601
+      xml.loc collection_url(collection)
+      xml.lastmod collection.updated_at.iso8601
     end
   end
-  @models.each do
+  @models.each do |model|
     xml.url do
-      xml.loc model_url(it)
-      xml.lastmod it.updated_at.iso8601
+      xml.loc model_url(model)
+      xml.lastmod model.updated_at.iso8601
     end
   end
 end

